src/Tightenco/Support/alias.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+$aliases = [
+    Tightenco\Contracts\Support\Arrayable::class => Illuminate\Contracts\Support\Arrayable::class,
+    Tightenco\Contracts\Support\Jsonable::class => Illuminate\Contracts\Support\Jsonable::class,
+    Tightenco\Support\Traits\Macroable::class => Illuminate\Support\Traits\Macroable::class,
+    Tightenco\Support\Arr::class => Illuminate\Support\Arr::class,
+    Tightenco\Support\Collection::class => Illuminate\Support\Collection::class,
+    Tightenco\Support\HigherOrderCollectionProxy::class => Illuminate\Support\HigherOrderCollectionProxy::class,
+];
+
+foreach ($aliases as $tighten => $illuminate) {
+    if (! class_exists($illuminate) && ! interface_exists($illuminate) && ! trait_exists($illuminate)) {
+        class_alias($tighten, $illuminate);
+    }
+}
```

Comment:
I'm wondering if we would be wiser to alias either _all_ classes, or _none_ of them if there's a single conflict. Theoretically, we could end up with a mixture of Laravel and Collect classes.

Reply:
I sort of doubt it, but I still like the idea of a single check instead of many.